cmd/jujud/agent/model/manifolds.go

```diff
@@ -187,13 +189,15 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 
 			NewFacade: singular.NewFacade,
 			NewWorker: singular.NewWorker,
+			// No Logger defined in singular package.
 		}),
 		// This flag runs on all models, and
 		// indicates if model's cloud credential is valid.
 		validCredentialFlagName: credentialvalidator.Manifold(credentialvalidator.ManifoldConfig{
 			APICallerName: apiCallerName,
 			NewFacade:     credentialvalidator.NewFacade,
 			NewWorker:     credentialvalidator.NewWorker,
+			Logger:        config.LoggingContext.GetLogger("juju.worker.credentialvalidator"),
```

Comment:
It would be nice if the logger name could be an exported const string from the package the logger relates to.